### DISCUSSION TITLE
where is the downloaded file stored

### DISCUSSION THREAD

**Discussion by VendingMp** [lnreader/lnreader#298] [Q&A] [answered]
I was wondering where my chapter file is saved

**Comment by rajarsheechatterjee** [comment, marked as answer]
Chapters are saved in a SQLite database, not as files.

**Reply by Dohko1** [reply]
Meaning we can't use a different app to read or play them with audio right? Because the app is perfect but I can't find an option to toggle audio speed

**Reply by rajarsheechatterjee** [reply]
Yeah, there's an open issue for text to speech enhancements.

**Reply by ai4hub** [reply]
why, thought I would be able to download as epub

**Reply by rajarsheechatterjee** [reply]
There's an option in the app for exporting downloaded chapters as an EPUB.

**Reply by Olowu12** [reply]
How

**Reply by JackDixon019** [reply]
> How

I know it's been months, but in case someone else also finds this thread: 
You can save downloaded chapters as an epub when viewing the novel in your library. The button to press is (at time of writing) at the top right corner of the app, next to the regular download button, and looks like a book with a bookmark. ![screenshot](https://github.com/user-attachments/assets/3f0faf3c-e056-4add-8216-3f6b8428fd9f)

**Comment by Olowu12** [comment]
I don't know if it's for different version but mine doesn’t have that. Please help

![Screenshot_20241223_104319_LNReader](https://github.com/user-attachments/assets/f351de32-6634-44dc-b853-031c3d3a0aaa)

**Reply by sthasam2** [reply]
Please check the Artifacts here.

https://github.com/LNReader/lnreader/actions/runs/13601051838#artifacts

There should be a zip file. Download it, extract it. Install the extracted apk.

There are no new releases, so you'll have to use these "builds"
